ide-common/src/main/kotlin/org/digma/intellij/plugin/navigation/codenavigation/CodeNavigator.kt

```diff
@@ -41,6 +42,10 @@ class CodeNavigator(val project: Project) {
         val spanIdWithoutType = CodeObjectsUtil.stripSpanPrefix(spanId)
 
         SupportedLanguages.values().forEach { language ->
+            //todo: Ask Shalom
+            if (ApplicationManager.getApplication().isUnitTestMode && language == SupportedLanguages.CSHARP) {
+                return@forEach
+            }
             val languageService = LanguageService.findLanguageServiceByName(project, language.languageServiceClassName)
```

Comment:
why is it necessary? if absolutely necessary please write a comment why

Reply:
in unit test mode, the call to LanguageService.findLanguageServiceByName with language CSHARP, crashes the plugin.